docs/pages/application-access/guides/jwt.mdx

```diff
@@ -0,0 +1,82 @@
+---
+title: Using Application Access JWT Tokens
+description: How to integrate use JWT tokens authentication with Application Access
+---
+
+# Integrating with JWTs
+
+Teleport includes a JWT token signed with Teleport's authority with each 
```

Comment:
I think I would be interested to learn more about valid use-cases for it. E.g. using it to get signed headers from teleport with roles and identity?